test/e2e-common.sh

```diff
@@ -93,16 +100,24 @@ function install_knative_serving() {
 # Parameters: $1 - Istio CRD YAML file
 #             $2 - Istio YAML file
 #             $3 - Knative Serving YAML file
+#             $4 - Knative Monitoring YAML file (optional)
 function install_knative_serving_standard() {
   INSTALL_ISTIO_CRD_YAML=$1
   INSTALL_ISTIO_YAML=$2
   INSTALL_RELEASE_YAML=$3
+  INSTALL_MONITORING_YAML=$4
+  if (( INSTALL_MONITORING )) && [[ -z "${INSTALL_MONITORING_YAML}" ]]; then
```

Comment:
Remove this condition. MONITORING_YAML is empty at this point. It will be available after you call build_knative_from_source in line 114.

Reply:
Removed.